contrib/blueflood-docker/configs/ES-Setup/init-es.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Yes! These are copied from blueflood-elasticsearch/src/main/resource/init-es.sh. Schema rarely changes; it's the blueflood.conf which people usually modify.